# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ecore/hotfix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/lib/ecore_evas/ecore_evas_x.c.orig	2010-10-06 15:30:48.000000000 -0400
+++ ./src/lib/ecore_evas/ecore_evas_x.c	2010-10-06 15:35:33.000000000 -0400
@@ -4468,7 +4468,6 @@
 #else
    return;
    ee = NULL;
-   win = 0;
 #endif   
 }
 
